dom/nodes/Document-createEvent.html

```diff
@@ -99,48 +101,56 @@
   "IDBVersionChangeEvent",
   "InstallEvent",
   "KeyEvent",
-  "MIDIConnectionEvent",
```

Comment:
Can you revert the changes to keep the ASCIIbetical order? This is what Emacs' sort-lines does, so it must be correct.